src/InstrumentationEngine.Attach/InstrumentationEngine.Attach.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project>
+  <PropertyGroup>
+    <SubComponent>Tools\Attach</SubComponent>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'EnlistmentRoot.marker'))\build\Managed.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AssemblyName>attach</AssemblyName>
```

Comment:
Did you want to make this name a little more descriptive? i.e. Microsoft.InstrumentationEngine.Attach.exe or something simliar?